packages/lib/src/components/internal/Address/constants.ts

```diff
@@ -1,2 +1,47 @@
 export const ADDRESS_SCHEMA = ['street', 'houseNumberOrName', 'postalCode', 'city', 'stateOrProvince', 'country'];
+const [STREET, HOUSE_NUMBER, POSTAL_CODE, CITY, STATE, COUNTRY] = ADDRESS_SCHEMA;
 export const COUNTRIES_WITH_STATES_DATASET = ['BR', 'CA', 'US'];
+
+// prettier-ignore
+export const ADDRESS_SCHEMAS = {
+    CA: [COUNTRY, HOUSE_NUMBER, STREET, [[CITY, 70], [POSTAL_CODE, 30]], STATE],
+    GB: [COUNTRY, [[HOUSE_NUMBER, 30], [STREET, 70]], [[CITY, 70], [POSTAL_CODE, 30]], STATE],
+    US: [COUNTRY, STREET, HOUSE_NUMBER, CITY, [[STATE, 50], [POSTAL_CODE, 50]]],
+    default: [COUNTRY, [[STREET, 70], [HOUSE_NUMBER, 30]], [[POSTAL_CODE, 30], [CITY, 70]], STATE]
+};
```

Comment:
Maybe we should consider adding and interface here

Reply:
Good idea!